arangosh/Import/QuickHistogram.h

```diff
@@ -182,14 +188,14 @@ class QuickHistogram : public arangodb::Thread {
         // old age string buffering & formatting
         char buffer[133];
         snprintf(buffer, sizeof(buffer),
-                 "%.3f,%.3f,%llu,%ld,%ld,%ld,%ld,%ld,%ld,%ld,%d,%s",
+                 "%.3f,%.3f,%llu,%ld,%ld,%ld,%ld,%ld,%ld,%ld,%lu,%s",
```

Comment:
As we are using the Logger anyway, why to we need `snprintf` here?
Right now there has been an adjustment to the format string, which may or may not extend the size of the generated output, but the buffer size has not been adjusted accordingly. The buffer size is 133 bytes, which to me is completely opaque. 
So why don't we use just the Logger here, which will solve all these potential issues?

Reply:
This was originally debug code using print to screen.  print to buffer was simplest conversion.

Reply:
I think using the Logger is even simpler, and as a side-effect, a lot less error prone and thus safer.

Reply:
There is a regex in collectd that is tuned to the above format.  Be a pain to duplicate in "<<" style.  Yes, buffer size should be bounced up, maybe 1024 to be safe.

Reply:
Sorry, but we shouldn't jeopardize security just because there is a regex in collectd.
By the way, I hope collectd also supports regexes like `([0-9]*\.?[0-9]+), so we could just repeat it the number of times we need it and be done without caring about the number of decimal places and the range of numbers.

Reply:
Is there a good reason not to use the json module in collectd in first place?

Reply:
I am not trying to build a socket connection to collectd.  Using it to simply pull data out of the existing log file.  This is already more code and more discussion than I intended for arangoimport.  Only wanted to be able to graph latencies and object counts.  Logging this as json would look horrid.